src/Savegame/SavedBattleGame.cpp

```diff
@@ -2698,6 +2699,40 @@ void SavedBattleGame::resetUnitHitStates()
 
 namespace
 {
+void flashMessageVariadicScript(SavedBattleGame* sbg, ScriptText message,...)
```

Comment:
Please do not use C variadic function, us C++ templated one (and add `Impl` to end of name to inform other that it should not be used directly).
Aside form that, this code can crash game, use `"{0}{1}{2}{3}{4}{5}"` with only one argument.

Reply:
I made some experiment: https://gcc.godbolt.org/z/Wxa2VQ

in `main` I added `z.add<void(*)(int, int), &Var>("aa");` that use new overload:
```cpp

	template<typename TX, TX X>
	void add(const std::string& func, const std::string& description = BindBase::functionWithoutDescription)
```

With this you can simplify definition by removing `*Arg` functions.